### ISSUE TITLE
Task list staging area for Google Code-In

### ISSUE THREAD

**Issue by SidharthBansal** [publiclab/plots2#3276]
### :point_right: Please append a comment `Added to GCI dashboard` after each task which you have posted so no one else opens the same task

**This is a staging area.**  Each task has (or will soon have) a link to a full issue including relevant lines of code, screenshots, and other helpful information as we prepare them for solving. Some are harder than others, but those tagged with `first-timers-only` are great to begin with.  (more [documentation on tasks](https://developers.google.com/open-source/gci/help/org-sample-tasks))

**Add new items:** Look here for new items to adapt for this list: [planning issues](https://github.com/publiclab/plots2/labels/planning), [break-me-up issues](https://github.com/publiclab/plots2/labels/break-me-up), [fto-candidates](https://github.com/publiclab/plots2/labels/fto-candidate). Fit tasks into the categories below.

**Format items to be descriptive, `[bracket otherwise]`:** As we create this list, it's OK to be brief, but if a task lacks description, tags, screenshots, and code links, `[Put it in brackets to indicate that it needs to be fleshed out]`

**Break out items here into their own issues:** Use the `gci-candidate` tag instead of the `gci` tag to manage the supply. We don't want to have more open issues than we can support at one time, and `gci` is what we'll direct students towards. `gci-candidate` is for "held back" issues we're saving up.

**Weighting:** some issues don't result in many lines of code, but are still big projects. Use tags `small`, `medium`, `big` when you'd like to help mark them ([read more](https://github.com/publiclab/plots2/issues/3284#issuecomment-422071860)) 

****

## Code: writing or refactoring

* [ ] Format inline thumbnail grids to follow new tag page styling (needs editing) https://github.com/publiclab/plots2/issues/5960
* [x] Remove asterisk from Boolean Full-Text Searches (#3490)
* [x] Turn off ip_address records in impressionist gem, switch to request_hash #3382
* [ ] display author profile bio below content in moderator email notifications: https://github.com/publiclab/plots2/blob/master/app/views/admin_mailer/notify_node_moderators.html.erb
* [x] move files from https://github.com/publiclab/plots2/tree/master/app/views/comment/ to https://github.com/publiclab/plots2/tree/master/app/views/comments/ to consolidate these very similar folders, and find any references to `/comment/` to change to `/comments/` -- issue at #3339 
- [x] Making a moving forward link or button on https://publiclab.org/stats page (described in #2650) -- `GCI dashboard`
- [x] Search Auto-complete not working on https://publiclab.org/post page (described in #2795) -- `GCI dashboard`
- [x] Views and number of comments of post is not seen on small screen (described in #2397) -- `GCI dashboard`
- [x] Change navbar links to icons (described in #3328 ) --- `GCI dashboard`
- [x] Add dropdown icons adjacent to navbar links where required (described in #3324 ) -- `GCI dashboard`
* [x] Trending topics on dashboard should not have repetitions. https://github.com/publiclab/plots2/issues/3188 -- GCI dashboard
* [ ] complete 'sortable headers' implementation following pattern in https://github.com/publiclab/plots2/issues/2352
* [ ] additional work on the Checklists system: https://github.com/publiclab/plots2/issues/2452
* [ ] calculate "answered" rate for questions posted in a certain time period (described in #2120)
* [x] Add a footer to the main page publiclab/image-sequencer#335
* [x] Display an installation message to user in the CLI publiclab/image-sequencer#338
* [x] Add a success prompt for saved sequence publiclab/image-sequencer#342
* [x] Add a success prompt for module installation from npm publiclab/image-sequencer#343
* [x] Display a flash message on removal of a step publiclab/image-sequencer#345
* [x] Include a back-to-top Button on scrolling down publiclab/image-sequencer#346
* [x] 'No file chosen' wrongly placed in Import Image module publiclab/image-sequencer#356
* [x] Arrange the modules in alphabetical order publiclab/image-sequencer#375
* [x] Rename omniauth.md #3731 `GCI dashboard`
* [x] Allow upload of only image files https://github.com/publiclab/image-sequencer/issues/429

**These can be moved up once they have code links or issue links:**

* [ ] `needs-code-links` Preserve redirect to original destination when accessing `/post` while not logged in (follow these steps: 1. publiclab.org/post You will be redirected to login page 2. Login 3. Redirected to dashboard -- Expected Behavior Redirected to /post.)
* [ ] display of an SVG at the URL `/profile/status/_ID_` for a `user` with `id` of `_ID_`, showing the `user.status` as one of these two SVGs: https://gist.github.com/jywarren/ea0523a09f6f8660486b14eeea15de95
* [ ] display if someone's been banned or is still active in moderator emails (https://github.com/publiclab/plots2/blob/master/app/views/admin_mailer/notify_node_moderators.html.erb) using the SVG at the above URL so moderators can tell if something's already been moderated without opening a link  -- issue at #3338 
* [ ] make username/email form appear side by side with Oauth user registration at https://publiclab.org/signup
* [ ] `needs-code-links` Remove extra space at bottom of /post
![image](https://user-images.githubusercontent.com/20972099/45542650-2300dc00-b830-11e8-89df-d66847e43dad.png)
* [x] `needs-code-links` Removal of 1 horizontal line
![image](https://user-images.githubusercontent.com/20972099/45542757-665b4a80-b830-11e8-9fca-d432dcff0397.png)
* [ ] `needs-code-links` Add generic video icon here, instead of being specific to YouTube. We should give equal priority to all the hosts. (https://github.com/publiclab/PublicLab.Editor/)
![image](https://user-images.githubusercontent.com/20972099/45542852-a9b5b900-b830-11e8-9101-b5085c6565d3.png)
* [ ] On clicking on '?' page should be opened in new tab.  Currently, it is opening on the same page, using `target="_blank"` on the `<a>` tag
![image](https://user-images.githubusercontent.com/20972099/45542912-d79afd80-b830-11e8-95d9-4bbfb533c1cd.png) -- `GCI dashboard`
* [ ] `needs-code-links` Similar behaviour (needs to open on a new page) reported for `Open Source Your Work` link at https://publiclab.org/signup and /post
* [ ] `needs-code-links` Hover text should redirect to page listing all the wiki articles and research notes with text `` water quality``
![image](https://user-images.githubusercontent.com/20972099/45543039-35c7e080-b831-11e8-8d25-bc060801a490.png)
* [ ]  `needs-code-links`Hover text should redirect to page listing all the wiki articles and research notes with text ``balloon-mapping``
![image](https://user-images.githubusercontent.com/20972099/45543048-382a3a80-b831-11e8-8f1b-e10dc81ee4de.png)
* [x]  `needs-code-links` Redundant `` Dashboard`` in the drop down menu #3327
![image](https://user-images.githubusercontent.com/20972099/45543183-96571d80-b831-11e8-85e9-722515ea1e56.png)
* [ ] `needs-code-links` Clearly mention that on RHS we have Research notes. `Liked by ____` appears ambiguous to me a lot
![image](https://user-images.githubusercontent.com/20972099/45543333-ec2bc580-b831-11e8-82f1-4e3e216d1914.png)
* [ ] `needs-code-links` sort wiki page likes according to date/time
![image](https://user-images.githubusercontent.com/20972099/45543424-06fe3a00-b832-11e8-9ec9-32e564306ee6.png)
* [ ] `needs-code-links` If a person has no barnstars and they click on the nav bar. There must be a message to show them. Just to enhance the readability
![image](https://user-images.githubusercontent.com/20972099/45543539-5d6b7880-b832-11e8-8b2b-33cdf3c28ebf.png)
* [ ] On removal of any of these tags, all three must be removed ![image](https://user-images.githubusercontent.com/20972099/45543633-9c99c980-b832-11e8-9e50-0c52fa207118.png)
* [ ] Same button three times to post work on same web page ![image](https://user-images.githubusercontent.com/20972099/45563088-e4881300-b869-11e8-87e2-c9108fb6f5c9.png)
* [ ] Same thread is placed repeatedly on the dashboard page due to multiple comments.
![image](https://user-images.githubusercontent.com/20972099/45541563-1464f580-b82d-11e8-98ca-c5b7759389ad.png)
I will suggest that one post should be shown only once. 
* [x] Use of common functions in /maps page : https://github.com/publiclab/plots2/issues/3316
* [ ] Standardize display of `n` Wiki and `m` Research Notes #2193
* [ ] When a person submits an empty sign up form.
![image](https://user-images.githubusercontent.com/20972099/44982930-14473900-af95-11e8-8588-f84b35cb0491.png) 
Error should also come that the fields are empty  #3340 -- `GCI dashboard`
* [ ] Test for the the above task
* [ ] Tag the author on the home page where their articles are written in grid view/List view (needs own issue)
* [ ] logging users to wiki page when a user logs in via header on wiki page (needs own issue)
* [ ] Clicking on like button if not logged in should redirect to login page. (needs own issue)
* [ ] Pagination broken on home page
![image](https://user-images.githubusercontent.com/20972099/45540558-62c4c500-b82a-11e8-92a9-53dfe375afbc.png)
see that url shows page 2 but the pagination is of page 1.  (needs own issue)
* [ ] `needs-code-links` make tags in `Suggested tags` on publiclab.org/post appear in tag input box when clicked - https://github.com/publiclab/PublicLab.Editor/

## Documentation/Training: creating/editing documents and helping others learn more

* [x] Write a page describing different tags used at github repositories like GCI, outreach, etc #2341 -- `GCI dashboard`
* [x] write a document explaining how to start contributing at public lab for Google Summer Of Code #3319 -- `GCI dashboard`
* [ ] write a document explaining how to start contributing at Public Lab for RGSoc (needs own issue)
* [ ] write documentation on how to open a Pull Request for first - timer contributor (needs own issue)
* [ ]  the table of contents on the github.com/publiclab/plots2 is inconsistent with the content present after it (needs own issue)
* [x] Add a list of available modules to the documentation publiclab/image-sequencer#337

## Outreach/Research: community management, marketing, or studying problems and recommending solutions

* [ ] Reporting Bugs on PL /Reporting Suggestions #3460 -- `GCI dashboard`
* [ ] Create new `first-timers-only` issues (https://publiclab.org/n/13667) -- `GCI dashboard`
* [ ] Convert `fto-candidates` into `first-timers-only` issues by embedding the friendly message which you can find in `first-timers-only` issues. `fto-candidate` issues can be found at https://github.com/publiclab/plots2/labels/fto-candidate -- `GCI dashboard`
* [ ] embed our welcome page in full at https://publiclab.org/code (needs own issue)
* [ ] read through welcome page https://publiclab.github.io/community-toolbox/ and highlight gaps or ambiguities (needs own issue in https://github.com/publiclab/community-toolbox/issues)
* [x] Refactor https://publiclab.github.io/community-toolbox/ to ensure list of contributors shows on multi-repository page (complex) - https://github.com/publiclab/community-toolbox/issues/29
* [ ] Help add code links to existing issues (described in #3424) -- `GCI dashborad`
* [x] Update welcome message in a pull request with asking a twitter handle of that user (described in #3485) -- `GCI dashboard`
* [x] Find a bug -- Google code-in publiclab/image-sequencer#350

## Quality Assurance: testing and ensuring code is of high quality

* [ ] #2151 - Fix "Show 0 more" on inline people lists 
* [x] #2150 - `Show more` should change to `Show less` on clicking -- GCI dashboard
* [ ] Read through Code Climate "smells" and open issues related to solving them with code links included and recommendations: https://codeclimate.com/github/publiclab/plots2/issues?severity%5B%5D=critical&severity%5B%5D=major&status%5B%5D=&status%5B%5D=open&status%5B%5D=confirmed&engine_name%5B%5D=structure&engine_name%5B%5D=duplication (described in #3706) -- `GCI dashboard`.

## Design: graphic design or user interface design
* [x] Exploring layouts for the dashboard #3355 instance count 50
* [ ] BIG TASK: Wiki page should be rendered on same page. On clicking on wiki pages nav bar on /notes/popular a new page opens up. But wiki pages should be rendered under the wiki nav bar. #3456
* [ ] FTO:  make the rsvp links in the events on the home page non clickable.
* [x] add hover text in [public lab editor](https://github.com/publiclab/PublicLab.Editor/)
![image](https://user-images.githubusercontent.com/20972099/44983562-3d68c900-af97-11e8-98a7-e6bf22956075.png) https://github.com/publiclab/PublicLab.Editor/issues/156
* [ ] ![image](https://user-images.githubusercontent.com/20972099/45532373-6bf56800-b811-11e8-9be0-ba0aa16d1905.png)
''Read More'' should be center aligned
* [ ] "Read more" and the next grid item should have hr in between
![image](https://user-images.githubusercontent.com/20972099/45532417-92b39e80-b811-11e8-9f86-b69bc9b417c6.png)
* [x] Implement "grid view" for inline lists, in addition to existing "table/list view" - https://github.com/publiclab/plots2/issues/1097
* [x] some Bootstrap 4 pre-upgrade work - https://github.com/publiclab/plots2/issues/1716
* [ ] [multiple-user]Copy welcomebot and first-timers-only templates into other projects - https://github.com/publiclab/plots2/issues/3681 -- `GCI dashboard`

## Looking for GSoC
**Reviewing PRs and collaboration efforts will be judged in your upcoming GSoC proposals**
If you are a GSoC'19 aspirant, then we need to tell you a good news. All the reviewing tasks will be judged in the upcoming GSoC'19 proposals. If you can review the PRs, help each other across issues, prs, create ftos and much more. Just commenting on the issues to help other folks is okay.
We will be having a special REVIEW/TEAMWORK section in the GSOC'19 proposal this summer.
All the best!!!

**Comment by SidharthBansal** [comment]
@jwarren I have a cool idea. I don't know it's feasibility level. Please help @publiclab/soc.

We have pending projects/new projects at PL. We will have mentors for GCI. We all want to do projects as many as we can to learn. So, I want to map these things together.

Like in GSOC and RGSOC, we can assign a project to a team of mentors(team of mentors can include even one mentor) and the project is completed by the students. We can apply the same approach here at GCI. But with one twist. Many students will be contributing to the same project instead of one. Any student can do any amount of work in any project. The project will be completed within the period of GCI by the students of GCI under the assistance of mentors. The mentors can break that project into smaller issues and subissues. These issues will be solved by the students. Students will get scores. Which will ultimately decide the top 6 winners of the GCI.
This will have many benefits:
1) Mentors can enhance managerial skills and can build more projects on production
2) GCI
3) Most importantly the public lab will grow its contributor population and its code will be enhanced.

**Comment by SidharthBansal** [comment]
@jywarren these are some of the issues which I came across while using PL as a developer.

**Comment by jywarren** [comment]
Ah, just a note on `Removal of image from the footer as we are removing localisation and internationalisation` -- we're not removing it, but we are going to be working on it more in #3262 

Almost caught up on all this, thanks for your patience!

**Comment by sagarpreet-chadha** [comment]
Hi @SidharthBansal...I added some tasks .
I was wondering if we should break these tasks especially UI Design into sub-tasks .

I guess we can break them into many sub parts . What do you think ?

**Comment by jywarren** [comment]
Just adding a few here that can be merged with the above:

* [ ] make username/email form appear side by side with Oauth user registration
* [ ] display of an SVG at the URL `/profile/status/_ID_` for a `user` with `id` of `_ID_`, showing the `user.status` as one of these two SVGs: https://gist.github.com/jywarren/ea0523a09f6f8660486b14eeea15de95
* [ ] display if someone's been banned or is still active in moderator emails (https://github.com/publiclab/plots2/blob/master/app/views/admin_mailer/notify_node_moderators.html.erb) using the SVG at the above URL so moderators can tell if something's already been moderated without opening a link

These could all use some code links!

**Comment by jywarren** [comment]
Some that could be broken out of `break-me-up` issues  -- a LOT of good material there -- (https://github.com/publiclab/plots2/labels/break-me-up):

* [ ] additional work on the Checklists system: https://github.com/publiclab/plots2/issues/2452
* [ ] some Bootstrap 4 work? https://github.com/publiclab/plots2/issues/1716

**Comment by SidharthBansal** [comment]
Go ahead @sagarpreet edit the list.
@jywarren we can add the tasks into the list by Editting the list this way we will not have long conversation in this issue 
Thanks all

**Comment by SidharthBansal** [comment]
@sagarpreet-chadha go ahead and write down the tasks as many as you can. We need atleast intial 150 tasks and atleast 25 open issue till december for gci. Also, we have to keep in mind the elderly fellows who want to contribute so we need even more tasks.

**Comment by SidharthBansal** [comment]
* [ ] follow these steps: 
1. publiclab.org/post You will be redirected to login page
2. Login
3. Redirected to dashboard
Expected Behavior Redirected to /post.
* [ ] Remove extra space at bottom of /post
![image](https://user-images.githubusercontent.com/20972099/45542650-2300dc00-b830-11e8-89df-d66847e43dad.png)
* [ ] Removal of 1 horizontal line
![image](https://user-images.githubusercontent.com/20972099/45542757-665b4a80-b830-11e8-9fca-d432dcff0397.png)
* [ ] Add generic video icon here, instead of being specific to YouTube. We should give equal priority to all the hosts.
![image](https://user-images.githubusercontent.com/20972099/45542852-a9b5b900-b830-11e8-9101-b5085c6565d3.png)
* [ ] On clicking on '?' page should be opened in new tab.
![image](https://user-images.githubusercontent.com/20972099/45542912-d79afd80-b830-11e8-95d9-4bbfb533c1cd.png)
Currently, it is opening on the same page.
* [ ] Similar behaviour reported for ' Open Source Your Work'
* [ ] Hover text should redirect to page listing all the wiki articles and research notes with text `` water quality``
![image](https://user-images.githubusercontent.com/20972099/45543039-35c7e080-b831-11e8-8d25-bc060801a490.png)
* [ ] Hover text should redirect to page listing all the wiki articles and research notes with text ``balloon-mapping``
![image](https://user-images.githubusercontent.com/20972099/45543048-382a3a80-b831-11e8-8f1b-e10dc81ee4de.png)
* [ ] Redundant `` Dashboard`` in the drop down menu.
![image](https://user-images.githubusercontent.com/20972099/45543183-96571d80-b831-11e8-85e9-722515ea1e56.png)
* [ ] Clearly mention that on RHS we have Research notes. Liked by ____ appears ambiguous to me a lot
![image](https://user-images.githubusercontent.com/20972099/45543333-ec2bc580-b831-11e8-82f1-4e3e216d1914.png)
* [ ] sort wiki page likes according to date/time
![image](https://user-images.githubusercontent.com/20972099/45543424-06fe3a00-b832-11e8-9ec9-32e564306ee6.png)
* [ ] If a person has no barnstars and they click on the nav bar. There must be a message to show them
![image](https://user-images.githubusercontent.com/20972099/45543539-5d6b7880-b832-11e8-8b2b-33cdf3c28ebf.png)
Just to enhance the readability
* [ ] @gauravano do we still need 
![image](https://user-images.githubusercontent.com/20972099/45543592-83911880-b832-11e8-845d-850a17388777.png)
on profile page
* [ ] On removal of any of these tags, all three must be removed
![image](https://user-images.githubusercontent.com/20972099/45543633-9c99c980-b832-11e8-9e50-0c52fa207118.png)

**Comment by Souravirus** [comment]
- [ ] Auto-complete not working on 'post' page (described in #2795) 
- [ ] Views and number of comments of post is not seen on small screen (described in #2397)
- [ ] Making a moving forward link or button on https://publiclab.org/stats page (described in #2650)

**Comment by jywarren** [comment]
I thought we can post them in comments, but then before we submit, we can combine them all in the top and delete the comments. That way we can coordinate. 

OR maybe even better, we can have a new issue compiling everything and close this issue so the conversation is preserved? We can do this sunday night or monday.

**Comment by gauravano** [comment]
Should we also add tags with each task? @jywarren

**Comment by jywarren** [comment]
yes please!

**Comment by gauravano** [comment]
![digest_week](https://user-images.githubusercontent.com/20878070/45569530-633a7b80-b87d-11e8-81da-fc767f4dfd4d.png)

Hi @SidharthBansal actually this button is a part of Vidit's project, I guess, he placed it there for testing. @ViditChitkara do we need it anymore

**Comment by gauravano** [comment]
Just copying this link here for reference https://developers.google.com/open-source/gci/help/org-sample-tasks . Also, everyone we need to add category and tags as per this link.

**Comment by gauravano** [comment]
## Code
- [ ] Change navbar links to icons (described in #3328 )
- [ ] Add dropdown icons adjacent to navbar links where required (described in #3324 )

**Comment by jywarren** [comment]
I also want to highlight that some example tasks were like this: 

https://codein.withgoogle.com/archive/2015/organization/4777925899452416/task/4744391868547072/

```
Fix a bug in the editor Pe (#3)
There are several bugs reported for the programmer's editor Pe. Fork the Pe repository, pick an issue and fix it.

Submit your fix as pull request (see the HaikuPorter wiki which works pretty much the same).
```

This means that when we create the tasks, we can generate "multi-use" tasks that refer to an issue list, to save time filing new tasks. We should try to use tags to direct people to "first-timers" vs. "next-step" sized issues. Maybe we can have a task for "Complete your first issue at Public Lab" vs. "Take the next step with a more complex issue at Public Lab" and "Tackle a complex project at Public Lab"?

**Comment by SidharthBansal** [comment]
Hi Jeff...we already have a great mechanism for directing people towards heavier tasks. When the mentors feel like student has done enough task you generally guide folks to do help me tagged issues. Similarly we can guide gci folks.

**Comment by SidharthBansal** [comment]
I will say we can have two categories as you told in the above comment
1. Gci candidate for normal to big sized gci tasks
2. Gci candidate and first timer tasks these tasks for first timers gci folks.
This will reuse our existing labels at public labs.

**Comment by jywarren** [comment]
That sounds great. I'm also interested in if we could make a task to paste
links to lines of code for new issues, as a task itself?

**Comment by SidharthBansal** [comment]
Yeah if issue is fto and gci candidate labelled then we will paste the code links else if it is only gci Candidate we may or may not paste code links.

**Comment by namangupta01** [comment]
Hi guys! Sorry for not being active from past month....This sounds great that public lab is selected for Google-Code-in finally. I would create some issues from my work of reply-by-email to comment for Code-in tasks.
Thanks

**Comment by SidharthBansal** [comment]
Please add 
```We are preparing to participate in Google Code-in, and have reserved this issue for participants in GCI - but we'd love to have your help with another one! Please check out https://code.publiclab.org to see more.```
to each ``gci `` task which you create.
Thanks

**Comment by Souravirus** [comment]
Hi @publiclab/mentors I have created some tasks from the above task list in [Google Code-in dashboard.](https://codein.withgoogle.com/dashboard/tasks/?sp-order=-modified&sp-my_tasks=false). Please edit if there are some issues with the tasks and add other issues that are present here which are not added in the dashboard. 
Thank you!!

**Comment by gauravano** [comment]
Hi @publiclab/mentors you can use the above task list for opening tasks in GCI dashboard, please append `GCI dashboard` after the task which you have used so that no one else opens the same task. Thanks!

**Comment by tech4GT** [comment]
@Mridul97 Hey can you add image sequencer issues to the list as well!! It will be a big help, I've been super busy! Thanks!!

**Comment by Mridul97** [comment]
@tech4GT I think I cannot edit the above list. I tried to look for an option to edit the comment but didn't find one.

**Comment by tech4GT** [comment]
@Mridul97 I think we can leave them in a comment here and @SidharthBansal can add them to the list then. Is that OK? Thanks!

**Comment by SidharthBansal** [comment]
Perfect varun

**Comment by Mridul97** [comment]
Yes, that's great! 

> @Mridul97 I think we can leave them in a comment here and @SidharthBansal can add them to the list then. Is that OK? Thanks!

Yes, that's great! I will do that!

**Comment by Mridul97** [comment]
Please append these tasks to the list!
- [ ] Add a footer to the main page https://github.com/publiclab/image-sequencer/issues/335
- [ ] Add a list of available modules to the documentation https://github.com/publiclab/image-sequencer/issues/337
- [ ] Display an installation message to user in the CLI https://github.com/publiclab/image-sequencer/issues/338
- [ ] Add a success prompt for saved sequence https://github.com/publiclab/image-sequencer/issues/342
- [ ] Add a success prompt for module installation from npm https://github.com/publiclab/image-sequencer/issues/343
- [ ] Display a flash message on removal of a step https://github.com/publiclab/image-sequencer/issues/345
- [ ] Include a back-to-top Button on scrolling down https://github.com/publiclab/image-sequencer/issues/346
- [ ] Find a bug -- Google code-in https://github.com/publiclab/image-sequencer/issues/350
- [ ] 'No file chosen' wrongly placed in Import Image module https://github.com/publiclab/image-sequencer/issues/356
- [ ] Arrange the modules in alphabetical order https://github.com/publiclab/image-sequencer/issues/375

**Comment by Souravirus** [comment]
@Mridul97, I have added these tasks to the list. Please see I have put them in the right categories. Also, please ensure that these tasks are put in the GCI dashboard. Thank you!!

**Comment by SidharthBansal** [comment]
Thanks sourav for putting them into the list.

**Comment by Mridul97** [comment]
@Souravirus I will add these tasks in GCI dashboard as well:+1: Thanks!

**Comment by jywarren** [comment]
hey you all, this is awesome. Thanks Mridul97 that's a great list!

**Comment by milaaraujo** [comment]
Hey @SidharthBansal, can you add this task to the list?

* [ ] Remove asterisk from Boolean Full-Text Searches (https://github.com/publiclab/plots2/issues/3490)

I already put this in the GCI dashboard. Thanks!

**Comment by aSquare14** [comment]
@SidharthBansal  
#3359 Can I add this to the dashboard ?

**Comment by SidharthBansal** [comment]
No not at all

**Comment by SidharthBansal** [comment]
This is staging area, it is not to be published as a whole. We need to
publish issues written in this list by making them issue on github repo
first and then on the gci dashboard.


Does this makes sense?

**Comment by aSquare14** [comment]
Oh Now it does. I am sorry for the misconfusion.
ᐧ

**Comment by SidharthBansal** [comment]
Ok

**Comment by Mridul97** [comment]
I have added these tasks to the GCI Dashboard. But I am unable to write --gci dashboard in front of them in the above list as I do not have access to edit the list. So someone could please do it. Thanks!
- [ ] Add a footer to the main page https://github.com/publiclab/image-sequencer/issues/335
- [ ] Add a list of available modules to the documentation https://github.com/publiclab/image-sequencer/issues/337
- [ ] Display an installation message to user in the CLI https://github.com/publiclab/image-sequencer/issues/338
- [ ] Add a success prompt for saved sequence https://github.com/publiclab/image-sequencer/issues/342
- [ ] Add a success prompt for module installation from npm https://github.com/publiclab/image-sequencer/issues/343
- [ ] Display a flash message on removal of a step https://github.com/publiclab/image-sequencer/issues/345
- [ ] Include a back-to-top Button on scrolling down https://github.com/publiclab/image-sequencer/issues/346
- [ ] Find a bug -- Google code-in https://github.com/publiclab/image-sequencer/issues/350
- [ ] 'No file chosen' wrongly placed in Import Image module https://github.com/publiclab/image-sequencer/issues/356
- [ ] Arrange the modules in alphabetical order https://github.com/publiclab/image-sequencer/issues/375

**Comment by jywarren** [comment]
Hi, everyone - are all draft tasks ready to be published in the GCI dashboard? Should we just approve the first 50?

**Comment by Souravirus** [comment]
Hi @jywarren, I have modified some tasks, so they are more readable. And I had a doubt with this task https://codein.withgoogle.com/dashboard/tasks/6571718295420928/ . Is this task required to submit some kind of documents and if it is required where should they submit it? It should be clarified

**Comment by SidharthBansal** [comment]
Hi folks whenever you create a task please mention the public lab repo and issue number so that it can be backtraced. It is really difficult for students to find the repo and the issue number. They are really new to OSS and young too.

**Comment by jywarren** [comment]
@souravirus - i think we may need input from Kirtika on this one, as they created it. I'm not sure!

@sidharthBansal - agreed, we def. need to review the tasks, i see. Everyone -- please ensure your tasks have links to GitHub issues!

**Comment by SidharthBansal** [comment]
## Looking for GSoC
**Reviewing PRs and collaboration efforts will be judged in your upcoming GSoC proposals**
If you are a GSoC'19 aspirant, then we need to tell you a good news. All the reviewing tasks will be judged in the upcoming GSoC'19 proposals. If you can review the PRs, help each other across issues, prs, create ftos and much more. Just commenting on the issues to help other folks is okay.
We will be having a special REVIEW/TEAMWORK section in the GSOC'19 proposal this summer.
All the best upcoming GSoCers!!!

**Comment by SidharthBansal** [comment]
Few Guidelines:
If @publiclab/mentors your issue in not posted on the GCI dashboard then it will be due to one of the following reason
* Missing repository name
* missing issue number
*  Too less issue detail
* We cannot have tasks like teaching Ruby, teaching Nodejs, teaching grunt, etc. Tasks for making youtube videos for RSpec, Rails etc will not be accepted at all. Sorry. We can help the students by giving material to external resources but creation of such tasks does no good to PL
* You have created a first timer issue but you have not used first-timer-only template
* missing code links
* missing solution details. A lot of people has reported the error, like buttons are overlapping on the webpage. But they did not tell what the student has to do? We should not assume that students know what we want them to. We want either one line space between them or we want different colors to distinguish them is not mentioned in the issue.

**Comment by SidharthBansal** [comment]
Please see that the task is not posted by any other mentor before posting the task onto GCI dashboard.
Leave a comment on the issue whichever you have posted on the GCI dashboard if it has not been published yet or if you are creating a new issue.

**Comment by aashna27** [comment]
Hi @SidharthBansal  Can you add this task to the list, it has been added to gci dashboard. 
[ publiclab/image-sequencer#429](https://github.com/publiclab/image-sequencer/issues/429)

**Comment by SidharthBansal** [comment]
Hi aashna27 please create a task on the GCI dashboard and send me the link
to the issue, I will add.

**Comment by aashna27** [comment]
Hi @SidharthBansal  Can you add this task to the list, it has been added to gci dashboard. 
[ publiclab/image-sequencer#429](https://github.com/publiclab/image-sequencer/issues/429)

[Gci dashboard](https://codein.withgoogle.com/dashboard/tasks/5938670969290752/)

is this fine?

**Comment by jywarren** [comment]
Thanks to hard work by @SidharthBansal and others, we now have 75 tasks published on the GCI dashboard, enough for the competition to begin! 🎉 🎉

**Comment by jywarren** [comment]
Hi folks, just added this one which would make a good GCI issue if someone wants to add it to the dashboard (please leave a comment here if you do it!):

https://github.com/publiclab/plots2/issues/3831

**Comment by milaaraujo** [comment]
Hey folks, I think the task 'Install plots2 on local machine' needs more mentors and preferably someone with experience in solving problems with the project installation! :weary:

**Comment by milaaraujo** [comment]
Install plots2: #3840

**Comment by jywarren** [comment]
This list may also be a great source for new GCI issues: https://github.com/publiclab/plots2/issues/3331

**Comment by SidharthBansal** [comment]
I have added 10. Mentors are requested to draft the tasks as quickly as
possible.

**Comment by aashna27** [comment]
@SidharthBansal  please publish this https://codein.withgoogle.com/dashboard/tasks/6044626013126656/

**Comment by SidharthBansal** [comment]
@jywarren there are many issues which are reported in this issue but are not made into separate issues and are not been worked upon. One way is to close this issue. Other way is to keep this issue open for GCI'19. We can continue things here. What do you suggest?
Creating a new issue from this containing all remaining work will be a BIG trouble to anyone as no one is knowing all the repos at PL and has this much time to verify whether all reported subissues are resolved.
Suggestions are welcomed.
Thanks

**Comment by CodeSarthak** [comment]
Hi @SidharthBansal I think, us GCI participants (@oorjitchowdhary @tsparksh @JonathanXu1 @ValentinaTironi @okonek @kevinzluo ) will be more than happy to look through these tasks, see for which PRs have been made and make a new issue for GCI'19 or otherwise to help future participants. Cheers!

CC : @jywarren @publiclab/guides

**Comment by SidharthBansal** [comment]
Sarthak thanks for support. Are you willing to open up an issue listing all
the existing issues in form of checklist? Kevin is busy with school. Oorjit
is on off till 10 January. It will be great if you can do this before other
folks get free.

**Comment by gauravano** [comment]
Nice idea for making a separate checklist. And, thank you Sarthak for
support.

**Comment by CodeSarthak** [comment]
Thanks @SidharthBansal @gauravano 

Here it is : #4581

**Comment by CodeSarthak** [comment]
Hey @SidharthBansal Can you make @ValentinaTironi @okonek and @kevinzluo members of Public Lab, so that I can add them in the GCI 18 team I just made. Thanks.

**Comment by SidharthBansal** [comment]
Sorry, GCI'18 is completed now, so I don't think that there is any need of
such group. We can tag them with their tagnames.

**Comment by SidharthBansal** [comment]
@CodeSarthak what is the progress here. Please do it before 15 January to get points for this task.

**Comment by jywarren** [comment]
🎉
